docs/configuration/recording_rules.md

```diff
@@ -45,8 +45,11 @@ dashboards, which need to query the same expression repeatedly every time they
 refresh.
 
 Recording and alerting rules exist in a rule group. Rules within a group are
-run sequentially at a regular interval. The names of recording and alerting rules
+run sequentially at a regular interval. The names of recording rules
 must be [valid metric names](https://prometheus.io/docs/concepts/data_model/#metric-names-and-labels).
+The names of alerting rules must be
+[valid label values](https://prometheus.io/docs/concepts/data_model/#metric-names-and-labels)
+which permits spaces for example.
```

Comment:
We don't need this bit.

Reply:
Removed it. I added it in attempt to clarify why one name allows more freedom than the other.